proto/api/v2/report_service.proto

```diff
@@ -118,4 +126,11 @@ service ReportService {
             delete: "/v2/reports/cancel/{id}"
         };
     }
+
+    // Downloads a generated report for the given report id
+    rpc DownloadReport(DownloadReportRequest) returns (DownloadReportResponse) {
+        option (google.api.http) = {
+            get: "/v2/reports/jobs/{report_job_id}/download"
```

Comment:
minor nit: `/v2/reports/jobs/{id}/download`, the prefix URL component indicates the resource type.

(Without the `jobs` component, the URL param is justified.)